js/ui/drawer/ui.drawer.js

```diff
@@ -87,6 +76,14 @@ const Drawer = Widget.inherit({
         });
     },
 
+    _setDeprecatedOptions() {
+        this.callBase();
+
+        extend(this._deprecatedOptions, {
+            'target': { since: '20.1', message: 'This option is supported partially and the functionality controlled by it was not supposed to the Drawer widget.' }
```

Comment:
```suggestion
            'target': { since: '20.1', message: 'Functionality associated with this option is not intended for the Drawer widget.' }
```